### ISSUE TITLE
[caffee2] Windows build / 'metanet_pb2' (a circular import)  Anaconda

### ISSUE THREAD

**Issue by PrestigeDevop** [pytorch/pytorch#83379]
### 🐛 Describe the bug

I want to use [Detectron](https://github.com/facebookresearch/Detectron) for an old project .
 I have Anaconda ```conda 4.13.0```
 

```
UserWarning: Caffe2 support is not enabled in this PyTorch build. Please enable Caffe2 by building PyTorch from source with `BUILD_CAFFE2=1` flag.
  warnings.warn('Caffe2 support is not enabled in this PyTorch build. 
```

can't build PyTorch from source for various reasons 
I think that has nothing to do with the issue  
```
  from caffe2.proto import caffe2_pb2, metanet_pb2, torch_pb2
ImportError: cannot import name 'metanet_pb2' from partially initialized module 'caffe2.proto' (most likely due to a circular import) 
```

  I edit the script located in D:\Anaconda\lib\site-packages\caffe2\proto\__init__.py
  and run it then change it back to its original code and now somehow it doesn't install or output anything !!
  
 So it seems that Caffee2 is integrated into Pytorch and the API is changed .
 I need to run the script of which use Detectron and old api of caffee2 
 
when I do 
```
python -c 'from caffe2.python import core' 2>/dev/null && echo "Success" || echo "Failure"
```
it fails 
 ```
ModuleNotFoundError: No module named 'detectron' 
 ```

to put it simply I need to run this Project with Anaconda --

https://github.com/krematas/soccerontable

### Versions

I can't run this script
